src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -1682,18 +1683,20 @@ private XMPPError onSessionAcceptInternal(
         // Loop over current participant and send 'source-add' notification
         propagateNewSources(participant, sourcesAccepted);
 
-        // Signal sources which have been added or removed to the conference since the Jingle session was initiated.
-        ConferenceSourceMap remoteSourcesToAdd = participant.getPendingRemoteSourcesToAdd();
-        ConferenceSourceMap remoteSourcesToRemove = participant.getPendingRemoteSourcesToRemove();
-        if (!remoteSourcesToAdd.isEmpty())
+        for (SourcesToAddOrRemove sourcesToAddOrRemove : participant.clearQueuedRemoteSourceChanges())
         {
-            jingle.sendAddSourceIQ(remoteSourcesToAdd, jingleSession);
-        }
-        if (!remoteSourcesToRemove.isEmpty())
-        {
-            jingle.sendRemoveSourceIQ(remoteSourcesToRemove, jingleSession);
+            AddOrRemove action = sourcesToAddOrRemove.getAction();
+            ConferenceSourceMap sources = sourcesToAddOrRemove.getSources();
+            logger.info("Sending a queued source-" + action + " to " + participantId + ", sources:" + sources);
```

Comment:
Will this log as a `source-Add` and `source-Remove` (with the capital letters) because of how the enum is spelled?  Not the end of the world but a little weird.

Reply:
Yes, I was lazy :)